{"links": {"self": {"href": "data/repositories/ignitionrobotics/ign-transport/issues/113/comments/54152353.json"}, "html": {"href": "#!/ignitionrobotics/ign-transport/issues/113#comment-54152353"}}, "issue": {"links": {"self": {"href": "data/repositories/ignitionrobotics/ign-transport/issues/113.json"}}, "type": "issue", "id": 113, "repository": {"links": {"self": {"href": "data/repositories/ignitionrobotics/ign-transport.json"}, "html": {"href": "#!/ignitionrobotics/ign-transport"}, "avatar": {"href": "data/bytebucket.org/ravatar/{4249390a-7f55-404b-990d-817ba94cc7ac}ts=1533306"}}, "type": "repository", "name": "ign-transport", "full_name": "ignitionrobotics/ign-transport", "uuid": "{4249390a-7f55-404b-990d-817ba94cc7ac}"}, "title": "[Fedora KDE] Tests do not pass on fresh source install - Discovery fails"}, "content": {"raw": "Workaround: `IGN_IP=127.0.0.1`\r\n\r\nThis prevents the use of distributed computing environments.", "markup": "markdown", "html": "<p>Workaround: <code>IGN_IP=127.0.0.1</code></p>\n<p>This prevents the use of distributed computing environments.</p>", "type": "rendered"}, "created_on": "2019-10-01T11:48:22.505820+00:00", "user": {"display_name": "Til Hoff", "uuid": "{5b5196a3-571e-4e4c-bed3-6f72e4dabba8}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B5b5196a3-571e-4e4c-bed3-6f72e4dabba8%7D"}, "html": {"href": "https://bitbucket.org/%7B5b5196a3-571e-4e4c-bed3-6f72e4dabba8%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/1835281ac000de88eb0ed5f38cd82eaed=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsTH-5.png"}}, "nickname": "turakar", "type": "user", "account_id": "557058:0928ff78-c705-4502-bf3b-52e34e5508f2"}, "updated_on": null, "type": "issue_comment", "id": 54152353}